core/services/synchronization/telem/telem_functions_request.proto

```diff
@@ -5,6 +5,7 @@ option go_package = "github.com/smartcontractkit/chainlink/v2/core/services/sync
 package telem;
 
 message FunctionsRequest {
-  bytes request_id = 1;
-  repeated string domains = 2;
+  string request_id = 1;
+  string node_address = 2;
+  repeated string domains = 3;
```

Comment:
I'm assuming it's OK to change field ID as this data is not collected in prod anywhere yet...? (or you'll change both sides?).

Reply:
Right, this is not used anywhere yet, so this is safe to shuffle the fields.